python/rps.py

```diff
@@ -0,0 +1,34 @@
+import random
+
+def rock_paper_scissors():
+    turn_options = ['rock', 'paper', 'scissors']
+
+    while True:
+        player_turn = input("Your Move:") 
```

Comment:
```suggestion
        player_turn = input("Your Move: ") 
```

Reply:
I would also lowercase `player_turn` so if the player enters `Rock` or `rock` or RoCk`, they'll all work fine